lib/transport/tests/test_transport_socket_proxy.c

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2020-2023 One Identity
```

Comment:
The correct copyright for this file today is: `Copyright (c) 2020-2024 One Identity LLC.`